src/modules/auth/components/shared/PresentationSection.component.tsx

```diff
@@ -0,0 +1,20 @@
+import { SupportedPlatformsList } from '../SignUp/SupportedPlatformsList.component';
+import { PhoneMockup } from '../SignUp/PhoneMockup.component';
+
+export function PresentationSection(): JSX.Element {
```

Comment:
Also, we should only have one `shared` folder, since it is representing things that are shareable in the whole app. So move `PresentationSection` to `components` folder.